### PR TITLE
feat(public): added icon

### PR DESCRIPTION
**Pull request by DalderupMaurice**
**NOTE** Currently only works for windows - I am not sure how to get it working for MacOSX - Linux
Tried several tuts and followed this: https://www.electron.build/configuration/configuration#MetadataDirectories-buildResources but the icon remains the same

This is probably also related to the building issue `yarn dist` and `yarn build` which currently do not work. This might be needed to get it working for MacOSx - Linux.

## Description
The nOS logo instead of the electron logo in the app bar.

## Motivation and Context
Basic requirement

## How Has This Been Tested?
/

## Types of changes

- [ ] Chore (tests, refactors, and fixes)
- [x] New feature (adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] I have read the **CONTRIBUTING** document.

## Closing issues
/

### CODE REVIEW

**Comment by mhuggins** [comment]
I'm pretty sure that the icon will only show in Mac/Linux for built versions of the app.

File: public/electron.js [hunk 1]
```diff
@@ -41,9 +41,13 @@ const isMac = process.platform === 'darwin';
 
 function createWindow() {
   const framelessConfig = isMac ? { titleBarStyle: 'hidden' } : { frame: false };
+  const iconPath = path.join(
+    process.env.NODE_ENV === 'production' ? process.env.PUBLIC_URL : __dirname,
```
Comment (mhuggins) [inline public/electron.js]: There's an `isDev` variable already, we should use that here (unless there's a good reason not to).

File: package.json [hunk 2]
```diff
@@ -85,9 +85,9 @@
     "webpack-merge": "^4.1.2"
   },
   "build": {
-    "appId": "com.nos.electron",
+    "appId": "io.nos",
```
Comment (mhuggins) [inline package.json]: Should we leave this with `electron` on it?  Or maybe more appropriately `desktop`?  i.e.: `io.nos.desktop`

**Comment by DalderupMaurice** [comment]
Should be good to go @mhuggins 
Note that I've used `app.GetAppPath` for the icon path & `__dirname` for the loading screen
It's recommended to do it this way (found on stackoverflow) & it works both locally + when creating a build.

**Pull request merged**